apps/daffio/src/app/content/pwa/pages/pwa/pwa.component.html

```diff
@@ -0,0 +1,159 @@
+<daff-hero color="accent" class="hero" size="small">
+  <daff-container size="xl">
+    <div class="hero__grid">
+      <div class="hero__text">
+        <h1 daffHeroTitle>
+          Build the <span>next generation</span> of ecommerce stores.
+        </h1>
+        <h2 daffHeroSubtitle>
+          Progressive Web Apps take advantage of the best features of web and native applications.
+        </h2>
+      </div>
+      <img class="hero__image" src="/assets/why-pwa/pwa-illustration.svg" />
+    </div>
+  </daff-container>
+</daff-hero>
+
+<daff-callout layout="centered" class="features">
+  <daff-container size="md">
+    <h3 daffCalloutTitle>
+      A new level of quality to help scale your business
+    </h3>
+    <p daffCalloutSubtitle>
+      Progressive Web Apps are a new way to offer incredible mobile shopping experiences that are fast, reliable and accessible through the web.
+    </p>
+    <div class="features__grid">
+      <daff-feature mode="compact">
+        <img daffFeatureIcon src="/assets/why-pwa/fast.svg" />
+        <h4 daffFeatureTitle>
+          Fast
+        </h4>
+        <p daffFeatureSubtitle>
+          PWAs provide consistently fast experiences. Data caching allow users to browse applications with minimal load time.
+        </p>
+      </daff-feature>
+      <daff-feature mode="compact">
+        <img daffFeatureIcon src="/assets/why-pwa/integrated.svg" />
+        <h4 daffFeatureTitle>
+          Integrated
+        </h4>
+        <p daffFeatureSubtitle>
+          A user experience more integrated with the native platform &mdash; app icon for home screen access, push notifications, immersive full screen experience, and more.
+        </p>
+      </daff-feature>
+      <daff-feature mode="compact">
+        <img daffFeatureIcon src="/assets/why-pwa/reliable.svg" />
+        <h4 daffFeatureTitle>
+          Reliable
+        </h4>
+        <p daffFeatureSubtitle>
+          With the help of services workers, PWAs cache key assets and data on devices, allow applications to load and perform samlessly in slow to no connectivity.
```

Comment:
"With the help of services workers, PWAs cache key assets and data on devices, allow applications to load and perform samlessly in slow to no connectivity." 

-->

"With the help of service workers, PWAs can cache key assets and data on devices, allowing applications to load and perform seamlessly with no connectivity."